src/force_fields_input.F

```diff
@@ -2377,9 +2378,9 @@ SUBROUTINE read_nequip_data(nequip)
 
       CHARACTER(len=*), PARAMETER                        :: routineN = 'read_nequip_data'
 
-      CHARACTER(LEN=default_path_length)                 :: cutoff_str
-      INTEGER                                            :: handle
-      LOGICAL                                            :: found
+      CHARACTER(LEN=default_path_length)                 :: allow_tf32_str, config_str, cutoff_str
+      INTEGER                                            :: handle, i, idx, len_config
+      LOGICAL                                            :: allow_tf32 = .FALSE., found
```

Comment:
Initializing a variable at declaration time creates an implicit save in Fortran. See also https://cp2k.org/conv#c003.